pset-iterators.js

```diff
@@ -137,7 +160,13 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 5:')
 
 // Add your code below this line
+const livingPresidents = presidents.filter(president => {
+  if(president.death_year === null) {
```

Comment:
@anniecdavis27 nice job, similar to the comment above, you can avoid having to use an `if` statement here due to how `.filter()` and `.find()` work...they both look for the callback function to return a boolean (true or false) so you can simply write 

```
return president.death_year === null
``` 

to accomplish the same thing